kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/players/BaseAudioPlayer.kt

```diff
@@ -443,34 +477,4 @@ abstract class BaseAudioPlayer internal constructor(private val context: Context
             )
         }
     }
-
-    inner class MediaSessionListener : MediaSessionCompat.Callback() {
```

Comment:
We don't need to explicitly override media session events, since our `ForwardingPlayer` already intercepts events sent by both media session and notification.